### ISSUE TITLE
Issue when using tinkers construct 1.16

### ISSUE THREAD

**Issue by Demonic** [someaddons/performant_issues#104]
**To Reproduce**
1. Have a server with Tinkers 1.16 (You must build the current file)
2. Have two players log onto the server at a smeltery each (2 separate smelteries)
3. Have player 1 begin smelting an iron block inside of smeltery let finish.
4. Once player 1 has finished smelting. Begin player 2 smelting copper block.
5. Have player 1 begin to pour iron block into basin
6. Once player 1 is done have player 2 pour copper into basin.
7. Player 1 should get iron block
8. Player 2 should get iron block as well..

This issue only appears when using performant on the server.

**Expected behavior**
Player 2 should've received copper.

**Comment by someaddons** [comment]
not seeing a 1.16 release somewhere yet

**Comment by Demonic** [comment]
> not seeing a 1.16 release somewhere yet

It's still in development. You need to build from their github. Bug still exists as of their latest code. Only occurs with performant installed.

**Comment by KnightMiner** [comment]
Actual release was today, see also SlimeKnights/TinkersConstruct#4283

**Comment by Darkfiend009** [comment]
Can confirm this is happening as well.

**Comment by MarioSMB** [comment]
As a temporary workaround, server admins can set cacheRecipes to false in Performant's configuration. The visual is still a bit strange, but at least the correct items are produced.

**Comment by Demonic** [comment]
> As a temporary workaround, server admins can set cacheRecipes to false in Performant's configuration. The visual is still a bit strange, but at least the correct items are produced.

This is good information. Appreciate the research.

**Comment by KnightMiner** [comment]
Any update on this? I'd rather not have to add to my mods documentation messages warning that performant will break my recipes, up to 10 duplicates of this issue already.

**Comment by Demonic** [comment]
> Any update on this? I'd rather not have to add to my mods documentation messages warning that performant will break my recipes, up to 10 duplicates of this issue already.

Unfortunately the mod author is extremely slow to reply. I would personally proceed forward with a warning message. Personally I opened this issue BEFORE tinkers would release so that there could be one less issue for you all to deal with, however that's going on +2 months now.

**Comment by someaddons** [comment]
fixed
